overlord/devicestate/handlers_install.go

```diff
@@ -92,3 +114,15 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 
 	return nil
 }
+
+func checkEncryption(model *asserts.Model) (bool, error) {
+	// TODO:UC20: also check if TPM is available, and return an error if the device must be
+	//            encrypted but we don't have TPM support
+
+	// Force encryption regardless of grade for developer testing
+	if osutil.FileExists(filepath.Join(dirs.RunMnt, "ubuntu-seed", ".force-encryption")) {
```

Comment:
Samuele suggested to just always encrypt if there is a TPM even if we are in grade dangerous and later provide a way to allow unencrypted operation for grade < signed. This approach sounds good to me. If we change this PR accordingly It wil mean that this PR will fail until the initrd has caught up.

Reply:
We still need TPM availability detection to decide if we can encrypt, but we can set it to a constant while the detection code is not added. I'll keep the file presence test (in reverse) to allow non-encrypted operation if grade is not secured, but we can replace it for a different mechanism if a new idea arises.